lib/bandit/http1/adapter.ex

```diff
@@ -367,12 +373,28 @@ defmodule Bandit.HTTP1.Adapter do
     415 => "Unsupported Media Type",
     416 => "Requested range not satisfiable",
     417 => "Expectation Failed",
+    418 => "I'm a teapot",
+    421 => "Misdirected Request",
+    422 => "Unprocessable Entity",
+    423 => "Locked",
+    424 => "Failed Dependency",
+    425 => "Too Early",
+    426 => "Upgrade Required",
+    428 => "Precondition Required",
+    429 => "Too Many Requests",
+    431 => "Request Header Fields Too Large",
+    451 => "Unavailable For Legal Reasons",
```

Comment:
Not in 9110

Reply:
[RFC 7725](https://datatracker.ietf.org/doc/html/rfc7725#section-3)

Proposed, but common.